### PR TITLE
[#532] Fixes the demo install

### PR DESCRIPTION
**Pull request by justafish**
## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/532

### CODE REVIEW

**Comment by justafish** [comment]
Schemata stopped wrapping things in a `data` object

**Comment by dawehner** [comment]
I'll try to fix the remaining problems.

File: .circleci/config.yml [hunk 1]
```diff
@@ -160,11 +159,6 @@ jobs:
     steps:
     - fix-php-env
     - checkout
-    - run:
```
Comment (dawehner) [inline .circleci/config.yml]: Do you mind explaining why we stop generating the lock file? ```update --lock``` shouldn't update the core version at least.

Comment (dawehner) [reply .circleci/config.yml]: I thought we have that so we can increase version numbers inside the `admin_ui_support` module.

**Pull request merged**